src/targets/c#/restsharp.js

```diff
@@ -0,0 +1,51 @@
+'use strict'
+
+var util = require('util')
+
+var CodeBuilder = require('../../helpers/code-builder')
+
+module.exports = function (source, options) {
+  var opts = util._extend({
+    indent: '  '
```

Comment:
I don't think you need this, nor the `opts` object at all, neither in the CodeBuilder() constructor, as none of the generated code is indented.